nixos/modules/services/backup/burpClient.nix

```diff
@@ -0,0 +1,65 @@
+{ config, lib, networking, pkgs, ... }:
+
+with lib;
+
+let
+
+  cfg = config.services.burpClient;
+
+in
+
+{
+
+  ###### interface
+
+  options = {
+
+    services.burpClient = {
+
+      enable = mkOption {
+        default = false;
+        description = "Whether to enable burp.";
+      };
+
+      configFile = mkOption {
+        default = "/etc/burp/burp.conf";
```

Comment:
That would require me to implement a decent generator for two configuration files for the server implementation and another one for the client and that's not going to happen now with a major update for burp around the corner.